docs/src/userguide/language_basics.rst

```diff
@@ -274,10 +414,24 @@ parameters and a new reference is returned).
 
 The name object can also be used to explicitly declare something as a Python
 object. This can be useful if the name being declared would otherwise be taken
-as the name of a type, for example,::
+as the name of a type, for example,
 
-    cdef ftang(object int):
-        ...
+.. tabs::
+
+    .. group-tab:: Pure Python
+
+        .. code-block:: python
+
+            @cython.cfunc
+            def ftang(int: object):
+                ...
+
+    .. group-tab:: Cython
+
+        .. code-block:: cython
+
+            cdef ftang(object int):
+                ...
 
 declares a parameter called int which is a Python object. You can also use
 object as the explicit return type of a function, e.g.::
```

Comment:
there are no tabs below

Reply:
Here are no tabs because there is an issue in cython not allowing moving this code to pure python: #2529. I suppose we need a not there about it.

Reply:
Added note about bug.